src/modules/lyrics/lyrics.js

```diff
@@ -5,7 +5,29 @@ BetterLyrics.Lyrics = {
       const artist = e.artist;
 
       BetterLyrics.Utils.log(BetterLyrics.Constants.FETCH_LYRICS_LOG, song, artist);
+      async function fetchFromLrclib(song, artist) {
+        const lrclibUrl = `https://lrclib.net/api/get?track_name=${encodeURIComponent(song)}&artist_name=${encodeURIComponent(artist)}`;
```

Comment:
Let's move all the constants to constants.js and reference them here for better maintainability like we do for better-lyrics-api